src/test/java/jaiden/ui/ParserTest.java

```diff
@@ -0,0 +1,22 @@
+package jaiden.ui;
+
+import jaiden.command.*;
```

Comment:
Perhaps a wildcard should not be used in the import statement here?
```suggestion
import jaiden.command.*;
```